### DISCUSSION TITLE
How to added custom table border XHTML to DOCX

### DISCUSSION THREAD

**Discussion by PerlDeveloperSurya** [jgm/pandoc#10066] [General]
how to render the custom table border XHTML to DOCX

**Comment by lifeunleaded** [comment]
@PerlDeveloperSurya If I recall correctly, you can set up a table style in the reference docx named "Table". Export the default reference document, set your borders in that style, save it, and convert with `--reference-doc=myreferencedoc.docx`.

**Comment by jgm** [comment]
Pandoc just tries to convert the document's structure. It will ignore any styling information in your XHTML table, including CSS.

**Reply by simjak** [reply]
Can I convert XHTML to markdown or PDF?
@jgm 
Found this https://github.com/aksdb/pandoc-xhtml

**Comment by TomBener** [comment]
See this [issue comment](https://github.com/jgm/pandoc/issues/3275#issuecomment-369198726)
